assets/templates/dataset-filter/hierarchy.tmpl

```diff
@@ -22,14 +22,14 @@
         </div>
     </div>
 </div>
-<div class="adjust-font-size--16">
+<div class="adjust-font-size--18 line-height--32">
     <div class="page-content link-adjust">
         <div class="wrapper">
             <div class="col-wrap">
                 <div class="col col--md-50 col--lg-35 margin-left-md--1">
                     <form class="margin-bottom--2" action="{{.Data.SearchURL}}">
                         <div class="clearfix">
-                            <label for="search" class="block font-size--17 padding-bottom--2 font-weight-700">Search <span class="visuallyhidden">{{ .Data.DimensionName }}</span></label>
+                            <label for="search" class="block font-size--18 line-height--32 padding-bottom--2 font-weight-700">Search <span class="visuallyhidden">{{ .Data.DimensionName }}</span></label>
```

Comment:
Similarly here, you technically don't need to add `font-size--18` because you have set it in the div on line 25 using the `adjust-font-size` class (you can remove all places you've used `font-size--18` within that div)